package.json

```diff
@@ -66,56 +66,26 @@
 		"wildemitter": "^1.2.1"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.3",
-		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
-		"@babel/preset-env": "^7.14.4",
-		"@babel/runtime": "^7.14.0",
+		"@nextcloud/babel-config": "^1.0.0-beta.1",
 		"@nextcloud/browserslist-config": "^2.1.0",
-		"@nextcloud/eslint-config": "^1.2.0",
-		"@nextcloud/eslint-plugin": "^1.5.0",
+		"@nextcloud/eslint-config": "^5.1.0",
+		"@nextcloud/stylelint-config": "^1.0.0-beta.0",
+		"@nextcloud/webpack-vue-config": "^4.0.3",
 		"@vue/cli-plugin-unit-jest": "^4.5.13",
 		"@vue/cli-service": "^4.5.13",
 		"@vue/test-utils": "^1.2.0",
-		"babel-eslint": "^10.1.0",
-		"babel-loader": "^8.2.2",
 		"babel-loader-exclude-node-modules-except": "^1.2.1",
-		"babel-plugin-add-module-exports": "^1.0.4",
-		"css-loader": "^5.0.1",
-		"eslint": "^5.16.0",
```

Comment:
why are we getting rid of eslint ?

Reply:
Some reading: https://github.com/nextcloud/app-tutorial/pull/319 ;)